framework/scenario/tests/managed_decimal_test.rs

```diff
@@ -42,6 +43,43 @@ pub fn test_managed_decimal() {
     );
 }
 
+#[test]
+pub fn test_managed_decimal_mixed() {
+    let fixed =
+        ManagedDecimal::<StaticApi, NumDecimals>::from_raw_units(BigUint::from(100u64), 2usize);
+    let fixed_2 = ManagedDecimal::<StaticApi, ConstDecimals<2>>::from(BigUint::from(5u64));
```

Comment:
It's `ConstDecimals<U2>` now, test will fail

Mass replace `It's `ConstDecimals<2>` -> `ConstDecimals<U2>`